app/controllers/users/aal3_controller.rb

```diff
@@ -1,7 +0,0 @@
-module Users
-  class Aal3Controller < ApplicationController
```

Comment:
Looks like the last redirect to this was removed a couple years ago in #4014 so I deleted it